eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/EventMeshMQTTServer.java

```diff
@@ -0,0 +1,221 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.runtime.boot;
+
+
+import org.apache.eventmesh.common.config.CommonConfiguration;
+import org.apache.eventmesh.runtime.acl.Acl;
+import org.apache.eventmesh.runtime.configuration.EventMeshMQTTConfiguration;
+import org.apache.eventmesh.runtime.core.protocol.mqtt.exception.MqttException;
+import org.apache.eventmesh.runtime.core.protocol.mqtt.processor.ClientConnectProcessor;
+import org.apache.eventmesh.runtime.core.protocol.mqtt.processor.ClientDisConnectProcessor;
+import org.apache.eventmesh.runtime.core.protocol.mqtt.processor.HealthCheckProcessor;
+import org.apache.eventmesh.runtime.core.protocol.mqtt.processor.MqttProcessor;
+import org.apache.eventmesh.runtime.core.protocol.mqtt.processor.PublishProcessor;
+import org.apache.eventmesh.runtime.core.protocol.mqtt.processor.SubscrubeProcessor;
+import org.apache.eventmesh.runtime.core.protocol.mqtt.processor.UnSubscrubeProcessor;
+import org.apache.eventmesh.runtime.meta.MetaStorage;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.Executor;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import io.netty.bootstrap.ServerBootstrap;
+import io.netty.buffer.PooledByteBufAllocator;
+import io.netty.channel.ChannelFuture;
+import io.netty.channel.ChannelHandler.Sharable;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelInboundHandlerAdapter;
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.ChannelOption;
+import io.netty.channel.ChannelPipeline;
+import io.netty.channel.epoll.EpollServerSocketChannel;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.channel.socket.nio.NioServerSocketChannel;
+import io.netty.handler.codec.mqtt.MqttConnAckVariableHeader;
+import io.netty.handler.codec.mqtt.MqttConnectReturnCode;
+import io.netty.handler.codec.mqtt.MqttDecoder;
+import io.netty.handler.codec.mqtt.MqttEncoder;
+import io.netty.handler.codec.mqtt.MqttFixedHeader;
+import io.netty.handler.codec.mqtt.MqttIdentifierRejectedException;
+import io.netty.handler.codec.mqtt.MqttMessage;
+import io.netty.handler.codec.mqtt.MqttMessageFactory;
+import io.netty.handler.codec.mqtt.MqttMessageType;
+import io.netty.handler.codec.mqtt.MqttQoS;
+import io.netty.handler.codec.mqtt.MqttUnacceptableProtocolVersionException;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class EventMeshMQTTServer extends AbstractRemotingServer {
+
+    private final EventMeshMQTTConfiguration eventMeshMQTTConfiguration;
+
+    private final EventMeshServer eventMeshServer;
+
+    private final MetaStorage metaStorage;
+
+    private final Acl acl;
+
+
+    protected final transient Map<MqttMessageType, MqttProcessor> processorTable =
+        new ConcurrentHashMap<>(64);
```

Comment:
The class is not serializable, is the `transient` keyword redundant?

Reply:
Noted，please review again.